button-examples/Android/android-button-example.md

```diff
@@ -23,213 +21,387 @@ There are four types of buttons:
 
-### Making buttons accessible
-
-Android's button component APIs support labeling for accessibility. To use labels...
-
-For more guidance on writing labels, go to [our page on how to write a good accessibility label](https://material.io/design/usability/accessibility.html#writing).
-
-
 ## Text button
 
 [Text buttons](https://material.io/components/buttons/#text-button) are typically used for less-pronounced actions, including those located in dialogs and cards. In cards, text buttons help maintain an emphasis on card content.
 
-### Text button example
+### Text button examples
 
-Source code API:
+API and source code:
 
 * `MaterialButton`
     * [Class description](https://developer.android.com/reference/com/google/android/material/button/MaterialButton)
     * [GitHub source](https://github.com/material-components/material-components-android/blob/master/lib/java/com/google/android/material/button/MaterialButton.java)
 
 The following example shows a text button with a text label.
 
-!["Text button example for Android with purple text 'Text' over a white background."](assets/text-button.svg)
+!["Text button"](assets/text-button.svg)
 
+In the layout:
 ```xml
-    <Button
-        android:id="@+id/textButton"
-        style="@style/Widget.MaterialComponents.Button.TextButton"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:text="Text button"
-        ...
-    />
+<Button
+    android:id="@+id/textButton"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:text="Text button"
+    style="@style/Widget.MaterialComponents.Button.TextButton"
+/>
+```
+
+_**Note:** `<Button>` is auto-inflated as `<com.google.android.material.button.MaterialButton>` via `MaterialComponentsViewInflater` when using a non-Bridge `Theme.MaterialComponents.*` theme._
+
+In code:
+```kt
+textButton.setOnClickListener {
+    // Respond to button press
+}
 ```
 
+The following example shows a text button with an icon.
+
+!["Text button with icon"](assets/text-button-icon.svg)
+
+In the layout:
+```xml
+<Button
+    ...
+    app:icon="@drawable/ic_add_24dp"
+    style="@style/Widget.MaterialComponents.Button.TextButton.Icon"
+/>
+```
 
 ### Anatomy and key properties
 
-A text button has text and an optional icon.
+A text button has a text label, a transparent container and an optional icon.
 
-<details><summary>Expand for anatomy diagram and key text button properties</summary>
+<details>
+<summary><b>Anatomy diagram</b></summary>
+<br>
 
-![\<Placeholder diagram of text button attribute. Replace this text if/when there is an approved diagram\>](assets/text-button-diagram.png)
+![\<Placeholder diagram of text button anatomy. Replace this text if/when there is an approved diagram\>](assets/text-button-diagram.png)
 
-**1. Text button**
+</details>
 
-* A. Text label
-* C. Icon
+<details>
+<summary><b>Attributes</b></summary>
+<br>
 
+<details>
```

Comment:
I think we're still scrapping the nested expandable content.

Reply:
Ah yeah indeed, will fix this